src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  @include flex;
+  justify-content: space-between;
+
+  height: 60px;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
+  &__items {
+    margin-left: 20px;
+  }
+
+  &__items:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    position: relative;
+    @include flex;
+
+    height: 60px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $mainTtextColor;
+    font-size: 12px;
+    font-weight: 500;
+
+    // &:visited {
+    //   color: $mainColor;
+    // }
+
+    &:hover {
+      color: $mainColor;
+    }
+  
+    &.is-active {
+      color: $mainColor;
+     
+      &::after {
+        content: " ";
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        width: 100;
```

Comment:
no need to specify width here in case in future  you might change the active link to another one.